src/cfg.c

```diff
@@ -233,40 +233,51 @@ static int parse_tiles_specification(const char* const arg, int32_t * const ntil
   return 1;
 }
 
-static double parse_number_or_die(const char *numstr,double min, double max)                               
+static int parse_uint8(const char *numstr,uint8_t* number,int min, int max)                               
 {
   char *tail;
-  double d = strtod(numstr, &tail);
-  if (*tail)
-    fprintf(stderr,"Expected number\n");
-  else
-    return d;
-  exit(1);
-  return 0;
+  int d = strtod(numstr, &tail);
```

Comment:
Since the parsed number is assigned to an int, wouldn't it be better to just use `strtol` here?

Reply:
You are right, I change it right now.